### DISCUSSION TITLE
Why does Lightning AI Fabric gives warning to training with precision = "bf16-mixed" instead of "bf16"

### DISCUSSION THREAD

**Discussion by basujindal** [Lightning-AI/pytorch-lightning#18263] [DDP / multi-GPU / multi-node] [answered]
It gives the following warning: UserWarning: bf16 is supported for historical reasons but its usage is discouraged. Please set your precision to bf16-mixed instead! I have seen many articles training the model in pure bf16 and getting similar accuracy.

**Comment by justusschock** [comment, marked as answer]
Hi, previously, bf16 was a shorthand notation for bf16-mixed. To clearly distinguish between mixed precision and pure bf16 we introduced the notion of `bf16-mixed` and `bf16-true` while just `bf16` still is a deprecated alias for `bf16-mixed`

**Reply by basujindal** [reply]
Oh got it, thanks for the clarification.

**Reply by scissorstail** [reply]
During training, it would have been better if the warning about the 'bf16 notation' being deprecated was more clearly stated, advising to choose either 'bf16-mixed' or 'bf16-true'. I was under the misconception that 'bf16-true' was the one deprecated. I just realized this now.
